README.md

```diff
@@ -4,6 +4,8 @@
 
 This library wraps gNMI functionality to ease usage with Cisco implementations in Python programs. Derived from [openconfig/gnmi](https://github.com/openconfig/gnmi/tree/master/proto).
 
+This is not an officially supported Cisco product. The gNMI servers implemented on Cisco networking operating systems are supported, but client interfacing and usage is up to the development team - this is an attempt to ease that client development process.
```

Comment:
My suggestion:

** THIS LIBRARY IS NOT AN OFFICIALLY SUPPORT CISCO PRODUCT. **
This library is intended to streamline the development of gNMI clients for interacting with Cisco products.